locale/ar_IQ/countries.xml

```diff
@@ -0,0 +1,263 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE countries SYSTEM "../../dtd/countries.dtd">
+
+<!--
+  * locale/ar_IQ/countries.xml
+  *
+  * Copyright (c) 2013-2016 Simon Fraser University Library
+  * Copyright (c) 2000-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localized list of countries. Based on ISO 3166-1.
+  -->
+<countries>
```

Comment:
Several of the country names are not translated. Just making sure you are aware of that.